internal/handler/web/users.go

```diff
@@ -180,7 +181,7 @@ func (h *Handler) PostUsersInvite(ctx *gin.Context) {
 		return
 	}
 
-	if err := h.users.InviteUser(ctx, invite.Email, sponsor); err != nil {
+	if err := h.users.InviteUser(ctx, invite.Email, sponsor); err != nil && !strings.Contains(err.Error(), "does not exist") {
```

Comment:
why is a not-found error okay? 
(also should be `errors.Is` if it is needed)

Reply:
hmmm, I think possibly we could get away with it just being in `SendInvite`, I'll check

Reply:
nope, doesn't work just checking on the `SendInvite` side, I think because the error is returned at the top of that function with userData so never gets down to the `errors.Is` part